test_runner/src/main/kotlin/ftl/doctor/Doctor.kt

```diff
@@ -1,44 +1,64 @@
 package ftl.doctor
 
+import com.fasterxml.jackson.databind.JsonNode
 import com.google.common.annotations.VisibleForTesting
+import ftl.args.AndroidArgsCompanion
 import ftl.args.ArgsHelper
 import ftl.args.IArgs
+import ftl.config.loadAndroidConfig
+import ftl.config.loadIosConfig
+import ftl.util.FlankFatalError
 import ftl.util.loadFile
 import java.io.Reader
+import java.lang.StringBuilder
 import java.nio.file.Path
 
-object Doctor {
-    fun validateYaml(args: IArgs.ICompanion, data: Path): String {
-        if (!data.toFile().exists()) return "Skipping yaml validation. No file at path $data"
-        return validateYaml(args, loadFile(data))
-    }
+fun validateYaml(args: IArgs.ICompanion, data: Path): String {
```

Comment:
This is the only a proposal but maybe better
WDYT? 
```kotlin

fun validateYaml(args: IArgs.ICompanion, data: Path): String =
    if (!data.toFile().exists()) "Skipping yaml validation. No file at path $data"
    else validateYaml(args, loadFile(data)) + preloadConfiguration(data, args is AndroidArgsCompanion)

```

Reply:
Sure, updated 👍